pkg/client/share_manager_client.go

```diff
@@ -0,0 +1,36 @@
+package client
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"google.golang.org/grpc"
+
+	rpc "github.com/longhorn/longhorn-share-manager/pkg/rpc"
+	"github.com/longhorn/longhorn-share-manager/pkg/types"
+)
+
+type ShareManagerClient struct {
+	Address string
+}
+
+func NewShareManagerClient(address string) *ShareManagerClient {
+	return &ShareManagerClient{
+		Address: address,
+	}
+}
+
+func (cli *ShareManagerClient) FilesystemTrim(isEncryptedDevice bool) error {
```

Comment:
Suggest renaming `isEncryptedDevice` to `encryptedDevice` instead. `is` is usually used for function or method name instead. (I know we had lots of variables named with the prefix `is`, but this should be improved over time by using the right convention)

Reply:
Addressed. Thank you.